rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/vendor/VendorController.java

```diff
@@ -274,6 +274,27 @@ public ResponseEntity<?> exportVendor(HttpServletResponse response) throws TExce
         return ResponseEntity.ok(HttpStatus.OK);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @Operation(
+            summary = "Merge two vendors.",
+            description = "Merge source vendor into target vendor.",
+            tags = {"Vendor"}
+    )
```

Comment:
Please document the possible response codes generated by this endpoint (which are 4 possible ways).

You can refer to other endpoints documenting the same.